### ISSUE TITLE
Idea: Create GitHub Actions schedule to pull new data and submit it

### ISSUE THREAD

**Issue by bramp** [ebelinski/apilevels#43]
It should be possible to create a github action to run one a week/month to download new data, and publish it, without human interaction.

Would this be useful?

**Comment by ebelinski** [comment]
Yes I think this would be a good idea! Do you have any thoughts on what it would take to achieve this?
